### DISCUSSION TITLE
Photo vs PDF

### DISCUSSION THREAD

**Discussion by MillerAnthony-FS** [community/community#132145] [GitHub Education] [answered]
### Hi everyone,

Greetings,

I am trying to apply for the Student program using iPhone and iPad. For providing documentation the website only allows my front facing cameras to work. It is awkward but I was able to, using iPhone, to capture my enrollment agreement from the screen of my iPad. I did the same for a Google Maps page showing how far I am from the University as I am an online student. I click the Process button, I wait, the web page reloads asking me to upload my documents again. Is there a way I can just use PDF files? Am I doing something wrong?

Thank you

**Comment by ghost** [comment]
Hello, type on the Internet, for example, “photo to PDF”, go to the first site and upload the photo, or you can use services like kappa, or, as a last resort, upload the photo in Word (sorry if I didn’t understand your problem correctly).

**Reply by MillerAnthony-FS** [reply]
I appreciate the response but it doesn't quite fit my need. I don't have a working printer so I cannot print out documents to hold up in front of the camera. This means I'm trying to use the camera to capture the display screen of the iPad and it isn't getting a clear enough image to post. Thank you though.

**Reply by ghost** [reply]
try using services that improve image quality (take a photo first and then try to improve the quality of the photo)

**Comment by danialtavakoli** [comment]
It sounds like you're encountering some challenges while trying to upload documents for your Student program application using your iPhone and iPad. Here are a few tips and suggestions that might help resolve your issue:

1. **PDF Files**: Yes, using PDF files can often be more reliable for uploading documents because they maintain formatting and are universally compatible. Here’s how you can convert your images to PDF on iPhone/iPad:

   - If you have captured images of your documents using your iPhone/iPad, you can convert them into a PDF file directly on your device. Here’s how:
     - Open the Photos app on your iPhone/iPad.
     - Select the images you captured (tap Select and choose multiple images).
     - Tap the Share button (the square with an arrow pointing up).
     - Scroll down and tap "Create PDF". This will combine the selected images into a single PDF file.
     - You can then save this PDF file to a location like Files app or iCloud Drive.

2. **Uploading Process**: When uploading the PDF files through the website, ensure the following:
   - Make sure the PDF files are saved locally on your iPhone/iPad (in Files app or iCloud Drive) before attempting to upload.
   - Navigate to the upload section of the website using Safari or another browser on your iPhone/iPad.
   - Select the option to upload files, and locate the PDF files you created.
   - Follow any prompts to complete the upload process.

3. **Website Compatibility**: Sometimes, certain websites or forms may not work perfectly on mobile devices or specific browsers. Here are some additional tips:
   - Ensure you are using a compatible browser like Safari or Chrome on your iPhone/iPad.
   - Try accessing the website from a desktop or laptop computer if possible, as they may provide a more stable and consistent experience for uploading documents.

4. **Technical Support**: If you continue to experience issues, consider reaching out to technical support or customer service for the Student program. They may have specific instructions or troubleshooting steps tailored to their website or application process.

By converting your images to PDF files and ensuring you are using a compatible browser, you should be able to upload your documents successfully. If the issue persists, contacting technical support for assistance would be the next best step.

**Comment by MillerAnthony-FS** [comment, marked as answer]
I finally went to my local library branch and had a document showing my enrollment and a Google map showing I live a state away from the university printed and I used those to take pictures. That worked.
